tensorflow_datasets/scripts/update_checksums.py

```diff
@@ -0,0 +1,108 @@
+# coding=utf-8
+# Copyright 2020 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Script to update all the checksums.tsv files with tabs instead of spaces"""
+
+import os
+from absl import app
+from typing import List
+
+import tensorflow.compat.v2 as tf
+
+_DATASET_TYPES = list(['audio',
+                        'image',
+                        'image_claffication',
+                        'onject_detection',
+                        'question_answering',
+                        'structured',
+                        'summarization',
+                        'text',
+                        'translate',
+                        'video'])
+
+_ROOT_DIR = os.path.normpath(os.path.dirname(os.path.dirname(__file__)))
+
+_TXT_CHECKSUM_DIR = os.path.join(_ROOT_DIR, 'url_checksums')
+_TXT_CHECKSUM_SUFFIX = '.txt'
+_TSV_CHECKSUM_SUFFIX = '.tsv'
+
+
+def get_tsv_checksums_paths() -> List[str]:
+  """Returns the paths of all the .tsv checksum files of folder datasets"""
+  paths = list()
+  for dataset_type in _DATASET_TYPES:
```

Comment:
To list datasets, how about using `tfds.list_builder` with `tfds.builder_cls(name)`, and `builder.code_path.parent / 'checksums.tsv'`